docs/client/session.md

```diff
@@ -0,0 +1,25 @@
+# Semantic conventions for session
+This document defines semantic conventions to apply to client-side applications when tracking sessions.
+
+A session is collection of Logs, Events, and Spans associated with a specific device or user within a specific period of time. 
+A session is represented by a unique identifier that is attached to Logs, Events, and Spans as an attribute.
+
+The primary purpose of sessions is to provide insight into the series of user actions or events that lead a critical or 
+interesting occurrence in the application's use. Sessions also provide a means to quantify overall application usage.
+
+
+## Attributes
+
+This attribute can be applied to all signals generated by an agent. It may be omitted from metrics, due to their aggregated nature. 
+
+| Attribute   | Type    | Description | Examples                             | Requirement level | 
+|-------------|---------|-------------|--------------------------------------|-------------------|
+| session.id  |  String | A GUID      | 00112233-4455-6677-8899-aabbccddeeff | required          |
+
+## How a session operates
+After a period of timeout, the `session.id` MUST be refreshed.
+The timeout period MUST be restarted when any Logs, Events, or Spans are recorded.
+The session timeout period CAN be customized.
+Default session timeout SHOULD be 30 minutes. This should only be done when the agent is configured, and MUST NOT be updated in the middle of a session.
+
```

Comment:
```suggestion
Default session timeout SHOULD be 30 minutes. Session timeout period SHOULD be established when the instrumentation is first configured and MUST NOT be updated in the middle of a session. 
```

Reply:
@breedx-splk it's a bit subtle but Session would be an SDK construct and instrumentations aren't aware of it, so this statement could read 'when the SDK is first configured'.